stories/core/ColorPicker.stories.tsx

```diff
@@ -0,0 +1,180 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React, { useState } from 'react';
+import {
+  ColorPicker,
+  ColorPickerProps,
+  IconButton,
+  Tooltip,
+} from '../../src/core';
+import { action } from '@storybook/addon-actions';
+import Color from '../../src/core/ColorPicker/Color';
+import { DefaultColors } from '../../src/core/ColorPicker/ColorPicker';
+
+export default {
+  component: ColorPicker,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    children: { control: { disable: true } },
+  },
+  title: 'Core/ColorPicker',
+} as Meta<ColorPickerProps>;
+
+export const Basic: Story<ColorPickerProps> = (args) => {
+  const { activeColorIndex = 0 } = args;
+
+  const [opened, setOpened] = useState(false);
+
+  const [currentColorName, setCurrentColorName] = useState(
+    DefaultColors[activeColorIndex].name,
+  );
+  const [currentColorValue, setCurrentColorValue] = useState(
+    DefaultColors[activeColorIndex].color,
+  );
+
+  const onClick = (index: number) => {
+    action(`Clicked color #${index}`)();
+    setCurrentColorName(DefaultColors[index].name);
+    setCurrentColorValue(DefaultColors[index].color);
+  };
+
+  return (
+    <>
+      <IconButton onClick={() => setOpened(!opened)}>
```

Comment:
White color swatch needs some border
![image](https://user-images.githubusercontent.com/83585998/133578831-c6a0aa6b-3768-414f-a164-afd478a710fa.png)
Hover: 
![image](https://user-images.githubusercontent.com/83585998/133578868-fcbcd34f-4cdf-4372-bf45-ec294457535f.png)

Reply:
I added a border around the color swatch in the Icon Button